src/app/features/creature/creature-template-resistance/creature-template-resistance.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component } from '@angular/core';
+
+import { CreatureHandlerService } from '../creature-handler.service';
+import { CreatureTemplateResistance } from '@keira-types/creature-template-resistance.type';
+import { CreatureTemplateResistanceService } from './creature-template-resistance.service';
+import { CREATURE_TEMPLATE_RESISTANCE_TABLE } from '@keira-types/creature-template-resistance.type';
+import { RESISTANCE } from '@keira-constants/options/creature-template-resistance';
+import { MultiRowEditorComponent } from '@keira-shared/abstract/components/editors/multi-row-editor.component';
+import { CREATURE_TEMPLATE_RESISTANCE_SCHOOL } from '@keira-constants/options/creature-template-resistance-school';
+
+@Component({
+  selector: 'keira-creature-template-resistance',
+  templateUrl: './creature-template-resistance.component.html',
+  styleUrls: ['./creature-template-resistance.component.scss']
+})
+export class CreatureTemplateResistanceComponent extends MultiRowEditorComponent<CreatureTemplateResistance> {
```

Comment:
@Kitzunu why multi-row? shouldn't be there only one row per creature? just like `creature_template` ?